app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt

```diff
@@ -50,14 +58,37 @@ class ReceiveTokensPresenter(
         load(isRefresh = true, scrollToUp = true)
     }
 
+    override fun onTokenClicked(tokenDataPayload: ReceiveTokenPayload) {
+        if (tokenDataPayload.containsInTwoNetworks) {
```

Comment:
existsInTwoNetworks

contains doesn't work in this semantics, we need passive form